app/src/main/java/com/kimdoori/githubapp/pullrequest/PullRequestActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.kimdoori.githubapp.pullrequest
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.kimdoori.githubapp.R
+import com.kimdoori.githubapp.databinding.ActivityPullrequestBinding
+
+class PullRequestActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityPullrequestBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setupBinding()
+        setupWebView()
+    }
+
+    private fun setupBinding() {
+        binding = DataBindingUtil.setContentView<ActivityPullrequestBinding>(
+            this, R.layout.activity_pullrequest
+        ).apply {
+            lifecycleOwner = this@PullRequestActivity
+        }
```

Comment:
```suggestion
        binding = DataBindingUtil
            .setContentView<ActivityPullrequestBinding>(this, R.layout.activity_pullrequest)
            .apply {
                lifecycleOwner = this@PullRequestActivity
            }
```
함수 체이닝 컨벤션을 지키면 좋을 것 같아요 😄